ext/FHistMakieExt.jl

```diff
@@ -119,7 +119,27 @@ function Makie.convert_arguments(P::Type{<:Stairs}, h::Hist1D)
 end
 Makie.convert_arguments(P::Type{<:Scatter}, h::Hist1D) = convert_arguments(P, bincenters(h), bincounts(h))
 Makie.convert_arguments(P::Type{<:BarPlot}, h::Hist1D) = convert_arguments(P, bincenters(h), bincounts(h))
-Makie.convert_arguments(P::Type{<:Errorbars}, h::Hist1D) = convert_arguments(P, bincenters(h), bincounts(h), binerrors(h)/2)
+
+function Makie.convert_arguments(P::Type{<:Makie.Errorbars}, h::FHist.Hist1D)
+    se = FHist.binerrors(FHist.sqrt, h)
+    bincounts = FHist.bincounts(h)
+    if all(==(1.0), FHist.sumw2(h))
+        # no weights used in filling
+        convert_arguments(P, FHist.bincenters(h), bincounts, se)
+    else
+        # weights used, switch to pearson error if error bar dips below zero
```

Comment:
I can't remember where I saw this, I vaguely remember it was from a pretty old document scan from possibly a national lab.

Or, from a email archive.

cc. @tamasgal @aminnj

Reply:
It feels vaguely familiar but I don't remember from where. It doesn't seem like an obvious approximation to other error bars we typically use. I even asked GPT4o if it was an approximation to something else and it just told me the obvious...

> ...The interval is centered around the square root of an adjusted sample size and extends symmetrically by 0.5 units above and below this center.